composer.json

```diff
@@ -1,53 +1,53 @@
-{
-    "name": "bowphp/framework",
-    "description": "The bow PHP Framework",
-    "keywords": ["framework", "bow"],
-    "license": "MIT",
-    "support": {
-        "issues": "https://github.com/bowphp/framework/issues",
-        "source": "https://github.com/bowphp/framework"
-    },
-    "require": {
-        "php": "^7.2|^8.0",
-        "bowphp/tintin": "^1.3",
-        "filp/whoops": "^2.1",
-        "nesbot/carbon": "^2.16",
-        "psy/psysh": "v0.10.*",
-        "fzaninotto/faker": "^1.8",
-        "pda/pheanstalk": "^4.0",
-        "neitanod/forceutf8": "^2.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^4.8",
-        "monolog/monolog": "^1.22",
-        "twig/twig": "^2.5",
-        "squizlabs/php_codesniffer": "3.*",
-        "aws/aws-sdk-php": "^3.87",
-        "phpstan/phpstan": "^0.12.87",
-        "php-amqplib/php-amqplib": "^3.0"
-    },
-    "authors": [
-        {
-            "name": "Franck DAKIA",
-            "email": "dakiafranck@gmail.com",
-            "role": "Lead Developer"
-        }
-    ],
-    "autoload": {
-        "files": [
-            "src/Support/helper.php"
-        ],
-        "psr-4": {
-            "Bow\\": "src/"
-        }
-    },
-    "suggest": {
-        "twig/twig": "For the use of template engine 'Twig'",
-        "papac/bow-blade": "For the use of template engine 'Blade'"
-    },
-    "scripts": {
-        "fix-sniffer": "phpcbf --standard=psr2 --severity=4 --tab-width=4 src tests",
-        "sniffer": "phpcs --standard=psr2 --severity=4 --tab-width=4 src tests",
-        "test": "phpunit --configuration phpunit.dist.xml"
-    }
-}
+{
+    "name": "bowphp/framework",
+    "description": "The bow PHP Framework",
+    "keywords": ["framework", "bow"],
+    "license": "MIT",
+    "support": {
+        "issues": "https://github.com/bowphp/framework/issues",
+        "source": "https://github.com/bowphp/framework"
+    },
+    "require": {
+        "php": "^7.2|^8.0",
+        "bowphp/tintin": "^1.3",
+        "filp/whoops": "^2.1",
+        "nesbot/carbon": "^2.16",
+        "psy/psysh": "v0.10.*",
+        "fzaninotto/faker": "^1.8",
+        "pda/pheanstalk": "^4.0",
+        "neitanod/forceutf8": "^2.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^8",
```

Comment:
- Upgrading the `PHPUnit` version.
- Using the `dos2unix` command to replace the end of file char `\r\n` with `\n` because of consistency.